src/readbin/.merlin

```diff
@@ -2,5 +2,6 @@ REC
 PKG ezjsonm
 PKG fileutils
 PKG re.posix
+PKG curl
```

Comment:
If in the future we are going to add an element to a list that appears sorted, may I propose that we keep the list sorted unless there is a good reason not to?

(There is no need to fix this now. One day we will do a housekeeping PR to fix things like this. But still, I feel it is worth pointing out the practice now to reduce the amount of future housekeeping needed.)